tests/wpt/metadata/dom/nodes/Document-contentType/contentType/contenttype_javascripturi.html.ini

```diff
@@ -1,5 +1,5 @@
 [contenttype_javascripturi.html]
   type: testharness
   [Javascript URI document.contentType === 'text/html']
-    expected: FAIL
+    expected: TIMEOUT
```

Comment:
An easy way to check is to make a build on a branch that does not include any of the changes in this PR and run the tests again.